ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -143,6 +143,10 @@ public class DrivetrainSubsystem extends SubsystemBase {
   private SwerveDrivePoseEstimator swervePoseEstimator = null;
   private Pose2d currentPosition = new Pose2d();
   private ArrayDeque<Pose2d> historicPositions = new ArrayDeque<Pose2d>(PositionHistoryStorageSize + 1);
+  // Standard deviations for poseEstimator updates
+  // The wpilib matrix constructor requires sizes specified as Nat types. 
+  private Matrix<N3,N1> visionStdDev = MatBuilder.fill(Nat.N3(), Nat.N1(), new double[]{0.7, 0.7, 10});
```

Comment:
Add a comment that these were found via experimentation.  Also, the vision x,y is more accurate than the rotation. So, for rotation, we want the navX to be trusted more than the vision.

Reply:
we didnt experiment tho

Reply:
these were the reccomended values